text/0119-instance-options.md

```diff
@@ -0,0 +1,459 @@
+
+**Environment variable**: `MEILI_SNAPSHOT_INTERVAL_SEC`
+**CLI option**: `--snapshot-interval-sec`
+**Default value**: `86400` (1 day)
+**Expected value**: an integer
+
+Defines the interval between each snapshot. Value must be given in seconds.
+
+#### 3.3.17. Import snapshot
+
+**Environment variable**: N/A
+**CLI option**: `--import-snapshot`
+**Default value**: `None`
+**Expected value**: a filepath pointing to a snapshot file
+
+Launches Meilisearch after importing a previously-generated snapshot at the given filepath.
+
+This command will throw an error if:
+
+- A database already exists
+- No valid snapshot can be found in the specified path
+
+This behavior can be modified with the `--ignore-snapshot-if-db-exists` and `--ignore-missing-snapshot` options, respectively.
+
+#### 3.3.18. Ignore missing snapshot
+
+**Environment variable**: N/A
+**CLI option**: `--ignore-missing-snapshot`
+**Default**: Disabled
+
+⚠️ This command-line option does not take any values. Assigning a value will throw an error.
+
+Prevents a Meilisearch instance from throwing an error when `--import-snapshot` does not point to a valid snapshot file.
+
+This command will throw an error if `--import-snapshot` is not defined.
+
+#### 3.3.19. Ignore snapshot if DB exists
+
+**Environment variable**: N/A
+**CLI option**: `--ignore-snapshot-if-db-exists`
+**Default**: Disabled
+
+⚠️ This command-line option does not take any values. Assigning a value will throw an error.
+
+Prevents a Meilisearch instance with an existing database from throwing an error when using `--import-snapshot`. Instead, the snapshot will be ignored and Meilisearch will launch using the existing database.
+
+This command will throw an error if `--import-snapshot` is not defined.
+
+#### 3.3.20. Memory usage
+
+**Environment variable**: `MEILI_MAX_MEMORY`
+**CLI option**: `--max-memory`
+**Default value**: 2/3 of the available RAM of the machine
+**Expected value**: an integer (`104857600`) or a human readable size (`100Mb`)
+
+Set the maximum size of the RAM used by Meilisearch. By default, Meilisearch adapts its behavior to make the indexation use a maximum two thirds of available resources.
+
+Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
+
+⚠️ WARNINGS
+- We do not recommend setting the full RAM size of your machine. For example, when running Meilisearch on a machine with 4GB of RAM, do not set this options to `4Gb`
+- This command-line option will not perfectly ensure the RAM usage, but will help you manage multiple Meilisearch engines on the same machine (for example, using Kubernetes). The search engine cannot guarantee the exact usage of the RAM.
+- If the number set is higher than the real available RAM in the machine, we cannot prevent Meilisearch from crashing.
+
+#### 3.3.21. Indexing jobs
+
+**Environment variable**: `MEILI_INDEXING_JOBS`
+**CLI option**: `--indexing-jobs`
```

Comment:
It is not very clear that this parameter is requiring a number of processors. A job for me is more a number of tasks, not correlated with any hardware.

Im not the best in hardware naming but these are more understandable for me: 
- `cores`
- `processors`
- `threads`

In elasticseach the setting is called [`processors`](https://octoperf.com/blog/2018/09/21/optimizing-elasticsearch/#detected-processors).

I would suggest something in the line of `indexing-processors`. 
Or but it might become too long `--indexing-number-processor` (not sure about `number`).

Reply:
only the term `threads` would be really accurate I think.

Reply:
Often the users in GitHub issues and discussions talk about thread as well

Reply:
`--indexing-threads` or `--indexing-number-threads` is also good for me!

Reply:
@gmourier if I set it to `5` does Meilisearch absolutely tries to use the 5 threads or does Meilisearch consider using them depending on the workload?

Reply:
No, it does not necessarily use all cores, --max-indexing-threads is a good option